scripts/signedbuildskadi.sh

```diff
@@ -140,7 +142,7 @@ echo TIMESKETCH_PASSWORD=$TIMESKETCH_PASSWORD >> ./.env
 echo POSTGRES_USER=$POSTGRES_USER >> ./.env
 echo POSTGRES_PASSWORD=$psql_pw >> ./.env
 echo NEO4J_PASSWORD=$neo4jpassword >> ./.env
-
+echo 
```

Comment:
What is this for?

Reply:
this was a misenter, will fix in next commit